custom_components/rivian/const.py

```diff
@@ -577,6 +583,55 @@
             icon="mdi:car-battery",
             entity_category=EntityCategory.DIAGNOSTIC,
         ),
+        RivianSensorEntityDescription(
+            key="limited_acceleration_cold",
+            field="limitedAccelCold",
+            name="Limited Acceleration (Cold)",
+            icon="mdi:snowflake-thermometer",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        RivianSensorEntityDescription(
+            key="limited_regen_braking_cold",
+            field="limitedRegenCold",
+            name="Limited Regenerative Braking (Cold)",
+            icon="mdi:snowflake-thermometer",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        RivianSensorEntityDescription(
+            key="bluetooth_front_fascia_hardware_failure_status",
+            field="btmFfHardwareFailureStatus",
+            name="Front Fascia Bluetooth Module Failure Status",
+            icon="mdi:bluetooth",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        RivianSensorEntityDescription(
+            key="bluetooth_rear_fascia_hardware_failure_status",
+            field="btmRfHardwareFailureStatus",
+            name="Rear Fascia Bluetooth Module Failure Status",
+            icon="mdi:bluetooth",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        RivianSensorEntityDescription(
+            key="bluetooth_instrument_controls_hardware_failure_status",
+            field="btmIcHardwareFailureStatus",
+            name="Instrument Controls Bluetooth Module Failure Status",
+            icon="mdi:bluetooth",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        RivianSensorEntityDescription(
+            key="bluetooth_right_front_door_hardware_failure_status",
+            field="btmRfdHardwareFailureStatus",
+            name="Right Front Door Bluetooth Module Failure Status",
+            icon="mdi:bluetooth",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        RivianSensorEntityDescription(
+            key="bluetooth_left_front_door_hardware_failure_status",
+            field="btmLfdHardwareFailureStatus",
+            name="Left Front Door Bluetooth Module Failure Status",
+            icon="mdi:bluetooth",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
```

Comment:
We should name all of these "Bluetooth Module Failure Status <type>" so they are grouped together similar to how we name other similar sensor types

Reply:
Makes sense, I stated that way then changed it. Not sure the thinking there.

Reply:
It makes more sense to read it this way, I agree. But we should remain consistent.